client/src/components/UnsavedChangesIndicator/UnsavedChangesIndicator.js

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import i18n from 'i18n';
+import classnames from 'classnames';
+
+const UnsavedChangesIndicator = ({
+  level = 'none',
+  minutes = {
+    notice: 5,
+    warning: 10,
+  },
+}) => {
+  if (level === 'none') {
+    return null;
+  }
+  const isWarning = level === 'warning';
+  const mins = isWarning ? minutes.warning : minutes.notice;
```

Comment:
I was surprised to see the value in the alert stayed stagnant. I was expecting it to count up as time passed. I might ignore the warning when it pops up at 20 minutes, but if I look down and see it has counted up to 5 hours I might go "oh shit I need to save".

People might also think it's a bug if it stays at the value.